src/communication/azext_communication/tests/latest/replacer.py

```diff
@@ -0,0 +1,145 @@
+# --------------------------------------------------------------------------
```

Comment:
can we name this recording_processors.py? it seems to be the convention from other CLIs
https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/tests/latest/recording_processors.py